### ISSUE TITLE
Bug: Ingame addons list feature shows incorrect addons enabled/disabled

### ISSUE THREAD

**Issue by Cruzix** [shagu/pfUI#211]
Ingame addons list feature shows incorrect addons enabled/disabled.

For people who want to reproduce, this feature is shown upon clicking on the FPS/MS panel module.

This has been an issue ever since this feature got released but I never got to report it

**Comment by Cruzix** [comment]
Heres a video showcasing the issue https://youtu.be/tinA5ogtUqg

**Comment by Road-block** [comment]
@shagu  I fixed this in my fork but I don't want to do a pull request because I coded it under the influence of 🍻  and the implementation is 💩  
The main issue was that the function that collects the addon information runs too early and some addons haven't yet loaded.  
The bad data is then preserved in the closure.  
Diff of [modules\addons.lua](https://github.com/shagu/pfUI/compare/master...Road-block:master#diff-daed487935fce180c00496448ae11989)  

I also made some small changes to the info panels

- Added a couple events to the durability (it didn't update on death)
- Added a tooltip handler to durability that shows the repair amount on mouseover.
- Added click handlers to Friends / Guild that open the Social frame to the correct tabs when clicked.

Diff of [modules\panel.lua](https://github.com/shagu/pfUI/compare/master...Road-block:master#diff-ed2c7311a934aa2d23cae7a82f62c585)
